lwa_antpos/reading.py

```diff
@@ -13,6 +13,24 @@
 # "data/LWA-352 Antenna Positions & System Status.xlsx"  # old name
 
 
+def read_antpos(source='yaml'):
```

Comment:
I'd recommend the docs specifying that the default is typically a good choice.

and either use `source=None` to denote default (aka let casey choose how I get them) or `default`. may be a lil clearer that way.

Make `read_antpos_xxx` start with underscore to denote privateness.

Reply:
agreed